### ISSUE TITLE
Fast breathing appearing twice

### ISSUE THREAD

**Issue by kaledip** [WorldHealthOrganization/smart-emcare-cmr#26]
![Screenshot_20230905_103425_EmCare dev](https://github.com/WorldHealthOrganization/smart-emcare-cmr/assets/122801361/1f5bd811-1536-4b6f-a7a8-16d3970c1fe6)

**Comment by ppanchal-argus** [comment]
@magp18 Please feel free to correct me if I am mistaken.
I checked the questionnaire @kaledip .
One of them is the boolean and other one is the Display item with same label and linkId : "fastbreathing-True"
Changes on L3 side are required to update the behavior.

Can change label to L4 only .

**Comment by magp18** [comment]
@ppanchal-argus Yes he had one for display and one for the calculation which was showing instead of being hidden. It's now fixed as per 0.0.129
